pkg/activator/stats_reporter.go

```diff
@@ -84,7 +84,7 @@ func NewStatsReporter(pod string) (*Reporter, error) {
 			Description: "Concurrent requests that are routed to Activator",
 			Measure:     requestConcurrencyM,
 			Aggregation: view.LastValue(),
-			TagKeys:     metrics.CommonRevisionKeys,
+			TagKeys:     append(metrics.CommonRevisionKeys, metrics.PodTagKey, metrics.ContainerTagKey),
```

Comment:
UT update?

Reply:
The existing test should cover after https://github.com/knative/pkg/pull/797 is in.